README.md

```diff
@@ -56,17 +56,27 @@ grcov ~/Documents/FD/mozilla-central/build -t coveralls -s ~/Documents/FD/mozill
 
 ## Build & Test
 
-In order to build, the LLVM libraries and headers are required.
+In order to build, either LLVM 7 or LLVM 8 libraries and headers are required. If one of these versions is sucessfully installed, build with:
 
 ```
 cargo build
 ```
 
-To run tests:
+And to run tests:
```

Comment:
This change is unrelated, can you revert it?

Reply:
Ok, done.

Reply:
I'm sorry for this, I must have not seen it :(